GVRf/Framework/framework/src/main/java/org/gearvrf/GVRScene.java

```diff
@@ -156,17 +156,27 @@ public int removeSceneObjectsByName(final String name) {
     /**
      * Remove all scene objects.
      */
-    public synchronized void removeAllSceneObjects() {
+    public synchronized void removeAllSceneObjects()
+    {
         final GVRCameraRig rig = getMainCameraRig();
         final GVRSceneObject head = rig.getOwnerObject();
         rig.removeAllChildren();
 
         NativeScene.removeAllSceneObjects(getNative());
-        for (final GVRSceneObject child : mSceneRoot.getChildren()) {
+        for (final GVRSceneObject child : mSceneRoot.getChildren())
+        {
             child.getParent().removeChildObject(child);
         }
 
-        if (null != head) {
+        mSceneRoot = new GVRSceneObject(getGVRContext());
```

Comment:
What is this change for? I think it might reverting a fix in master.

Reply:
@NolaDonato i believe 'git blame' indicated that you changed this code.  do you have insight on the change?